cli/install/install.go

```diff
@@ -446,8 +531,9 @@ func installTt(binDir string, installCtx InstallCtx, distfiles string) error {
 	if err != nil {
 		return err
 	}
+
```

Comment:
Why empty line here? Defer is related to the code above, not the line below.

Reply:
fixed